tests/conftest.py

```diff
@@ -830,188 +263,17 @@ def archive():
-    weight = pathlib.Path( FileOnDiskMixin.local_path ) / "test_data/test_ztf_image.weight.fits"
-    flags = pathlib.Path( FileOnDiskMixin.local_path ) / "test_data/test_ztf_image.flags.fits"
-    sources = pathlib.Path( FileOnDiskMixin.local_path ) / "test_data/test_ztf_image.sources.fits"
-    psf = pathlib.Path( FileOnDiskMixin.local_path ) / "test_data/test_ztf_image.psf"
-    psfxml = pathlib.Path( FileOnDiskMixin.local_path ) / "test_data/test_ztf_image.psf.xml"
-    return image, weight, flags, sources, psf, psfxml
-
-
-@pytest.fixture
-def example_ds_with_sources_and_psf( example_image_with_sources_and_psf_filenames ):
-    image, weight, flags, sources, psf, psfxml = example_image_with_sources_and_psf_filenames
-    ds = DataStore()
-
-    ds.image = Image( filepath=str( image.relative_to( FileOnDiskMixin.local_path ) ), format='fits' )
-    with fits.open( image ) as hdul:
-        ds.image._data = hdul[0].data
-        ds.image._raw_header = hdul[0].header
-    with fits.open( weight ) as hdul:
-        ds.image._weight = hdul[0].data
-    with fits.open( flags ) as hdul:
-        ds.image_flags = hdul[0].data
-    ds.image.set_corners_from_header_wcs()
-    ds.image.ra = ( ds.image.ra_corner_00 + ds.image.ra_corner_01 +
-                    ds.image.ra_corner_10 + ds.image.ra_corner_11 ) / 4.
-    ds.image.dec = ( ds.image.dec_corner_00 + ds.image.dec_corner_01 +
-                     ds.image.dec_corner_00 + ds.image.dec_corner_11 ) / 4.
-    ds.image.calculate_coordinates()
-
-    ds.sources = SourceList( filepath=str( sources.relative_to( FileOnDiskMixin.local_path ) ), format='sextrfits' )
-    ds.sources.load( sources )
-    ds.sources.num_sources = len( ds.sources.data )
-
-    ds.psf = PSF( filepath=str( psf.relative_to( FileOnDiskMixin.local_path ) ), format='psfex' )
-    ds.psf.load( download=False, psfpath=psf, psfxmlpath=psfxml )
-    bio = io.BytesIO( ds.psf.info.encode( 'utf-8' ) )
-    tab = votable.parse( bio ).get_table_by_index( 1 )
-    ds.psf.fwhm_pixels = float( tab.array['FWHM_FromFluxRadius_Mean'][0] )
-
-    return ds
-
-
-@pytest.fixture
-def example_source_list_filename( example_image_with_sources_and_psf_filenames ):
-    image, weight, flags, sources, psf, psfxml = example_image_with_sources_and_psf_filenames
-    return sources
-
-
-@pytest.fixture
-def example_psfex_psf_files():
-    psfpath = ( pathlib.Path( FileOnDiskMixin.local_path )
-                / "test_data/ztf_20190317307639_000712_zg_io.083_sources.psf" )
-    psfxmlpath = ( pathlib.Path( FileOnDiskMixin.local_path )
-                   / "test_data/ztf_20190317307639_000712_zg_io.083_sources.psf.xml" )
-    if not ( psfpath.is_file() and psfxmlpath.is_file() ):
-        raise FileNotFoundError( f"Can't read at least one of {psfpath}, {psfxmlpath}" )
-    return psfpath, psfxmlpath
+@pytest.fixture( scope="module" )
+def catexp(data_dir, persistent_dir):
```

Comment:
Is this not something that makes sense to move into one of the fixture subfiles?

Reply:
I split the fixtures based on survey (mostly) so this Gaia catalog doesn't really belong with any of them. 

That said, it may belong because it targets a field which corresponds to one of the surveys' test images. I'll check.